acceptance/openstack/networking/v2/extensions/attributestags_test.go

```diff
@@ -48,4 +49,9 @@ func TestTags(t *testing.T) {
 	tags, err = attributestags.List(client, "networks", network.ID).Extract()
 	th.AssertNoErr(t, err)
 	th.AssertDeepEquals(t, tags, []string{})
+
+	// Verify the tags are also set in the object Get response
+	gnetwork, err := networks.Get(client, network.ID).Extract()
+	th.AssertNoErr(t, err)
+	th.AssertDeepEquals(t, gnetwork.Tags, []string{"a", "b", "c"})
```

Comment:
This must be some kind of bug with `AssertDeepEquals` - I'm not sure why it's passing. 

If you change this to:

```go
t.Logf("Network tags: %#v", gnetwork.Tags)
th.AssertEquals(t, len(gnetwork.Tags), 3)
```

You'll see that no tags exist and the assert now fails. But if you move these lines to before the tags are deleted (line 46), all is good.

Reply:
> You'll see that no tags exist and the assert now fails. But if you move these lines to before the tags are deleted (line 46), all is good.

Ouch sorry about that, rebase mistake - as you say it does pass which is strange, I'll see if I can figure out why.